### ISSUE TITLE
Are *Wrap classes just Decorating Envelopes? If yes, let's rename and adjust them

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/takes#915]
I think all `*Wrap` are just [decorating envelopes](https://www.yegor256.com/2017/01/31/decorating-envelopes.html). So, let's renaming them properly and change them (if necessary) to work exactly as a decorating envelopes.

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this issue

**Comment by paulodamaso** [comment]
@fabriciofx Yes, they're decorating envelopes, so what changes are you proposing exactly?

**Comment by fabriciofx** [comment]
@paulodamaso the idea is rename all `*Wrap* classes to use the same naming as Cactoos: Envelope. So, for example, `TkWrap` will becomes `TkEnvelope` and will be abstract class too.
